src/main.rs

```diff
@@ -35,7 +35,7 @@ struct IndexArgs {
 
     /// File descriptor to read triples from.
     /// Defaults to `stdin`.
-    #[arg(default_value = "-")]
```

Comment:
**question:** is there any issues with having input as positional argument? i.e.

`index data.nt`
instead of
`index -i input.nt`

Also this would set `-i` as the short option, which kind of clashes with index in `pseudo -i`

Reply:
ah jeah. that might be better. Make `--input` and `--index`. I am anyway not a big fan of using abreviations. 
or `-i` and `-x` for index. @cmdoret : can you do a PR on this?